README.rst

```diff
@@ -276,9 +234,10 @@ Use this installation mode if you want the version from a particular GitHub bran
     python -m pip install git+https://github.com/NVIDIA/NeMo.git@{BRANCH}#egg=nemo_toolkit[all]
 
 
-From source
-~~~~~~~~~~~
-Use this installation mode if you are contributing to NeMo.
+NeMo GitHub Repository
```

Comment:
Id prefer From Source - this is developer nomenclature for building the repository from the repository

Reply:
"From source" is vague. How about "Build from source repository?"

Reply:
Change to Build from Source